internal/pubsub/driver/driver.go

```diff
@@ -70,6 +73,12 @@ type Topic interface {
 	// then the call to Close should proceed and the call to SendBatch
 	// should fail immediately after Close returns.
 	Close() error
+
+	// IsRetryable should report whether err can be retried.
+	// err will always be a non-nil error returned from SendBatch.
+	// If its second return value is non-zero, it is the suggested amount
+	// of time to sleep before retrying.
+	IsRetryable(err error) (bool, time.Duration)
```

Comment:
Suggest expanding to note that returning 0 is fine, a backoff will be used.

Do we even need to expose this? (i.e., why not always use backoff?).

Reply:
Some services might provide a suggested delay. E.g. see https://code.googlesource.com/gocloud/+/master/spanner/retry.go#130. But maybe this is rare enough that we can just ignore it? It is worth noting that the [gRPC retry proposal](https://github.com/grpc/proposal/blob/master/A6-client-retries.md#retry-policy-capabilities) doesn't allow for custom delays.

Reply:
WDYT? Happy to remove it.

Reply:
Let's keep it simple for now.

Reply:
Done.